ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -319,10 +322,15 @@
       "The key store certificate alias to be used for internal client requests. If not set, "
           + "the system will fall back on the Vert.x default choice";
 
-  public static final String KSQL_LOGGING_SKIP_RESPONSE_CODES_CONFIG =
-      KSQL_CONFIG_PREFIX + "logging.skip.response.codes";
-  private static final String KSQL_LOGGING_SKIP_RESPONSE_CODES_DOC =
-      "A list of response codes to skip logging";
+  public static final String KSQL_LOGGING_SKIPPED_RESPONSE_CODES_CONFIG =
+      KSQL_CONFIG_PREFIX + "logging.skipped.response.codes";
+  private static final String KSQL_LOGGING_SKIPPED_RESPONSE_CODES_DOC =
+      "A list of HTTP response codes to skip during server request logging";
+
+  public static final String KSQL_LOGGING_RATE_LIMITED_REQUEST_PATHS_CONFIG =
+      KSQL_CONFIG_PREFIX + "logging.rate.limited.request.paths";
```

Comment:
Same comment as above regarding clarifying that this a server request logging config, in contrast to other types of logging.